src/Sulu/Bundle/AdminBundle/Resources/js/utils/Ajv/keywords/nameDefinition.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type {KeywordDefinition} from 'ajv';
+
+const nameDefinition: KeywordDefinition = {
```

Comment:
not really sure what this does? can you give me a hint? 😇

Reply:
The json schema returned by the server often looks like this

~~~json
{
  "type": "object",
  "properties": {
    "foo": {
      "name": "foo",
      "type": "string"
    }
  }
}
~~~

But `name` is not a valid json schema keyword, so we need to explicitly allow it.

Reply:
In ajv v7, the strict mode is enabled by default, therefore this wasn't needed before. We could also disable that, but I think it's generally good to have strict mode enabled

Reply:
alright sounds good 👍